content/cloud/cloud-plugin-reference.md

```diff
@@ -1,24 +1,30 @@
-title = "Cloud Command Reference"
+title = "Fermyon Cloud Plugin Command Line Interface (CLI) Reference"
```

Comment:
For me, "Cloud Command Reference" or "Cloud CLI Reference" or such like remain easier because _this is the reference for the `spin cloud` command._  I feel like I know that it's Fermyon and I don't want to have to care that it's a plugin (that's an implementation detail of how we deliver it on the cloud cadence instead of the Spin cadence).  But if you're not keen on that, then could I maybe persuade you down to "Fermyon Cloud CLI Reference"?

Reply:
"Cloud Command Reference" it is :-)